index.html

```diff
@@ -122,43 +146,54 @@ <h2 class="text-primary  col-4">Prizing</h2>
         <div class="bg-light text-dark p-2">
             <br>
             <h2 class="text-center">Featured Speakers</h2>
```

Comment:
- [ ] Talking about "featured speakers" part. Is important to add the 6 elements. At same time, please, add the arrow to show more when you're on mobile size.
![image](https://user-images.githubusercontent.com/20563252/132961546-ebe8d617-54df-40ca-a501-d14612c4ec61.png)
![image](https://user-images.githubusercontent.com/20563252/132961554-e79ce013-29bc-430b-beb4-4e025d34583b.png)

- Hint: I understand the template could show the home page on mobile size doesn't have a footer, but please, don't forget to add it.